### PR TITLE
Use default image for worker

### PR DESCRIPTION
**Pull request by johnSchnake**
The worker's default image was using a hardcoded
value and not referencing the const that was
defined for that purpose leading to the
aggregator and worker using different images.

* Notes for reviewer
A seemingly minor problem, I worry about what some of the edge cases might be if we have different defaults there. It could lead to some awkward behavior if we have a few point releases.

I _know_ we need to also add more tests but this is a tiny bug with an obvious cause. We need to spike eventually on improving testing of gen input/output.

Fixes: #643
Signed-off-by: John Schnake <jschnake@vmware.com>

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -297,8 +299,8 @@ func New() *Config {
 	}
 
 	// TODO (timothysc) reference the other consts
-	cfg.WorkerImage = "gcr.io/heptio-images/sonobuoy:latest"
-	cfg.ImagePullPolicy = "Always"
+	cfg.WorkerImage = DefaultImage
+	cfg.ImagePullPolicy = DefaultSonobuoyPullPolicy
```
Comment (stevesloka) [inline pkg/config/config.go]: Since the const `DefaultImage` uses `gcr.io/heptio-images/sonobuoy:buildinfo.Version`, I think the pull policy should change to `IfNotPresent` so that if it's already there, it won't always attempt to pull (especially in air-gapped envs).

Comment (johnSchnake) [reply pkg/config/config.go]: Done.
